src/App.tsx

```diff
@@ -45,8 +80,10 @@ class App extends React.Component<{}, State> {
     }))
   }
 
-  public setElection = (election: Election) => {
-    this.setState({ election })
+  public resetVotes = () => {
```

Comment:
is `resetVotes` meant to be resetting a single ballot's selections? If so, strongly recommend naming this `resetSelections` or `resetBallot`, otherwise it sounds like you're tracking the set of votes cast and this is how you remove all the votes.

Reply:
Sure, we can update this. I don't think we all have a shared understanding of what all the various terms mean. If the device doesn't have a feature to track all votes cast, then a function `resetVotes` would not be doing anything other than clearing the votes/selections of the current voter. At present this function resets the `votes` key to an empty object, thus the function clearly describes what it is doing. Let's do a full pass across all the function names when we're together next to make them as clear as possible.

Reply:
Doing a bit of refactoring… and I'm convinced. Changing `resetVotes` to `resetBallot`.